packages/chain-helpers/src/errorhandling/ExtrinsicError.ts

```diff
@@ -107,10 +107,10 @@ export const ExtrinsicErrors = {
  * PalletIndex reflects the numerical index of a pallet assigned in the chain's metadata.
  */
 export enum PalletIndex {
-  CType = 11,
```

Comment:
We had to change the pallet indices to be compliant with the current testnet. One should not change the order of pallets (e.g. indices for the pallets) in `construct_runtime`.